rc/trafficserver.service.in

```diff
@@ -27,8 +27,8 @@ Restart=on-failure
 RestartSec=5s
 LimitNOFILE=1000000
 ExecStopPost=/bin/sh -c ' \
-        export TM_PIDFILE=$(@exp_bindir@/traffic_layout} 2>/dev/null | grep RUNTIMEDIR | cut -d: -f2)/manager.lock ; \
-        /bin/rm -f $TM_PIDFILE ; \
```

Comment:
Just checking: are we sure we want to remove the -f? -f will return a non-zero return code in the case of a permissions error. If all we care about is that the pid file does not exist after this, regardless of whether it existed before or not, then -f is correct.

Or do we want a non-zero return code if the pid file did not exist? In that case, removing the -f is correct.

Reply:
We want the non-zero status code. I think we assume that ATS started up correctly in the first place.

Reply:
OK, that makes sense.